docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,142 @@
+* `Threat`: Specific threats the rule detects (such as Cobalt Strike or BPFDoor).
+* `Use Case`: The type of activity the rule detects and its purpose. Use cases include:
+** `Asset Visibility`: Detects changes to specified asset types.
+** `Configuration Audit`: Detects undesirable configuration changes.
+** `Guided Onboarding`: Example rule, used for {elastic-sec}'s guided onboarding tour.
+** `Identity and Access Audit`: Detects activity related to identity and access management (IAM).
+** `Log Auditing`: Detects activity on log configurations or storage.
+** `Network Security Monitoring`: Detects network security configuration activity.
+** `Threat Detection`: Detects threats.
+** `Vulnerability`: Detects exploitation of specific vulnerabilities.
+
+[float]
+[[select-all-prebuilt-rules]]
+=== Select and duplicate all prebuilt rules
+
+. Go to *Manage* -> *Rules*, then select the *Elastic rules* filter.
+. Click *Select all _x_ rules* above the rules table.
+. Click *Bulk actions* -> *Duplicate*.
+. Select whether to duplicate the rules' exceptions, then click *Duplicate*.
+
+You can then modify the duplicated rules and, if required, delete the prebuilt ones.
+
+[float]
+[[update-prebuilt-rules]]
+=== Update Elastic prebuilt rules
+
+Elastic regularly updates prebuilt rules to optimize their performance and ensure they detect the latest threats and techniques. When updated versions are available for your installed prebuilt rules, the *Rule Updates* tab appears on the *Rules* page, allowing you to update your installed rules with the latest versions.
+
+. Go to *Manage* -> *Rules*, then select the *Rule Updates* tab.
++
+NOTE: The *Rule Updates* tab doesn't appear if all your installed prebuilt rules are up to date.
++
+[role="screenshot"]
+image::images/prebuilt-rules-update.png[The Rule Updates tab on the Rules page]
+
+. Do one of the following:
+* Update all available rules: Click *Update all*.
+* Update a single rule: Click *Update rule* for that rule.
+* Update multiple rules: Select the rules and click *Update _x_ selected rule(s)*.
++
+TIP: Use the search bar and *Tags* filter to find the rules you want to update. For example, filter by `OS: Windows` if your environment only includes Windows endpoints. For more on tag categories, refer to <<prebuilt-rule-tags>>.
+
+[float]
+[[download-prebuilt-rules]]
+=== Download latest Elastic prebuilt rules
+
+As of {stack} >=7.13.0, you can download the latest version of Elastic prebuilt rules outside of a regular release cycle. This feature ensures you have the latest detection capabilities before upgrading to the latest {stack}.
+
+To download the latest version of prebuilt rules:
+
+. In {kib}, go to *Management* -> *Integrations*.
+. Search for "Prebuilt Security Detection Rules."
+. Select the integration, then select the *Settings* tab. The integration settings page is displayed.
++
+[role="screenshot"]
+image::images/install-prebuilt-settings.png[]
```

Comment:
I think this UI has changed: 
![image](https://github.com/elastic/security-docs/assets/137783811/e683bcb7-7a43-4c1e-af5f-4c49d703029f)

Reply:
^^ That's what the UI looks like when the rules are already installed, but for a brand-new deployment the current screenshot _almost_ matches the UI — the API reference tab is new, so I'll update with that.

Reply:
Ah, got it!